wiki/Performance_Troubleshooting/en.md

```diff
@@ -86,3 +86,22 @@ While this may seem like the best option (and indeed can be, if your hardware ca
 - GPUs generally aren’t meant to render 1000+ frames per second, so unexpected behaviour may occur, including lower performance or higher latency than otherwise attainable.
 
 Even so, if heat generation and potentially overworking your hardware is not considered an issue, unlimited may be the option that provides you the most stable frame times.
+
+### Custom
+
+This frame limiter attempts to keep a frame rate equal to your custom preference. However, this is a setting that cannot be changed to within the in-game menu.
+
+To change osu! custom maximum framerate, you'll need to tweak some values in the config file. Here's the steps:
```

Comment:
```suggestion
To change osu!'s custom maximum framerate, you'll need to tweak some values in the config file:
```
colon can't be used after incomplete clause

Reply:
Thank you for your review. In my language, colon can be used in the middle of the sentence when splitting into different routes to connect with the incomplete clause.
I'll fix these.